merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/AerieAppDriver.java

```diff
@@ -53,7 +50,7 @@ public static void main(final String[] args) {
     // Configure an HTTP server.
     final var javalin = Javalin.create(config -> {
       config.showJavalinBanner = false;
-      if (configuration.javalinLogging().isEnabled()) config.enableDevLogging();
+      config.enableDevLogging();
```

Comment:
The only thing that made me raise my eyebrows is the word "Dev" here, do you figure this is still appropriate for production?

Reply:
My understanding is that javalin allows you to set custom logging handlers for request events and websocket events, and `enableDevLogging` just sets them to a default. Without that, either javalin would produce no logs or I'd have to make custom handlers (and I don't think that would be useful).

This way, I use `enableDevLogging` to enable all output, and then filter out stuff below the logging level in slf4j.

Reply:
I'm assuming that this just enables debug-level logs within Javalin (see https://javalin.io/documentation#configuration):
> enable dev logging (extensive debug logging meant for development)

The actual level setting in the docker-compose file is the real gatekeeper here, so this just allows debug logs if/when you want to enable them. 

@JoelCourtney If you want you can probably wrap this in a conditional to check if the `slf4j` log level is debug or not. I think a `System.getProperty("org.slf4j.simpleLogger.defaultLogLevel").equalsIgnoreCase("debug")` may work?

Reply:
Since `System.getProperty()` can return a `null` you'd actually want:
```java
Optional.ofNullable(System.getProperty("org.slf4j.simpleLogger.defaultLogLevel")).ifPresent(p -> {
    if (p.equalsIgnoreCase("debug")) config.enableDevLogging();
}
```

Reply:
Or a simple `logger.isDebugEnabled() || logger.isTraceEnabled()`, sorry to complicate things above.

Reply:
`logger.isDebugEnabled()` is sufficient. Done. Thanks!